CoreFoundation/String.subproj/CFString.h

```diff
@@ -165,7 +165,7 @@ struct __CFConstStr {
         uint8_t _pad[4];
     } _base;
     uint8_t *_ptr;
-    uint32_t _length;
+    unsigned long _length;
```

Comment:
If it is needed for this platform to change then it should have an ifdef for gating the change only for that specific platform.